### PR TITLE
fixes to use asmt grade for titles

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: common/src/main/resources/templates/iab-body.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
                             <span class="label" th:text="#{report.iab.short-name}"></span><span class="label" th:text="#{'report.subject.' + ${assessment.subject} + '.name'}"></span>
                         </span>
                     </div>
-                    <div class="h4" th:text="#{('report.iab.title.grade.' + ${report.studentEnrollment.gradeCode})(${#strings.toString(exam.schoolYear)}, ${support.getLastTwoDigits(exam.schoolYear + 1)})}"></div>
+                    <div class="h4" th:text="#{('report.iab.title.grade.' + ${report.assessment.gradeCode})(${#strings.toString(exam.schoolYear)}, ${support.getLastTwoDigits(exam.schoolYear + 1)})}"></div>
```
Comment (wtritch) [inline common/src/main/resources/templates/iab-body.html]: Should this be 
```
${#strings.toString(exam.schoolYear-1)}, ${support.getLastTwoDigits(exam.schoolYear)})
```
?

**Comment by wtritch** [comment]
The `student-header` fragment is supposed to show the student's enrolled grade, correct?
We'll need translation values for "enrolled" and "assessment" on grades.  Right now, we don't have any translation values for grade 10, for instance:

```
report.grade.08=8
report.grade.08.label=G8
report.grade.08.color=green
report.grade.11=11
report.grade.11.label=G11
report.grade.11.color=maroon
```

File: common/src/main/resources/templates/iab-body.html (hunk 1 above)
Comment (esotrope) [reply common/src/main/resources/templates/iab-body.html]: yes it should - that is fixed in develop but not prod

Comment (wtritch) [reply common/src/main/resources/templates/iab-body.html]: I think so.  As long as we're deploying a reporting hotfix, we might as well fix any small known issues.  (Especially the obvious ones like the school year always being wrong.)

File: common/src/main/resources/templates/iab-body.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
                             <span class="label" th:text="#{report.iab.short-name}"></span><span class="label" th:text="#{'report.subject.' + ${assessment.subject} + '.name'}"></span>
                         </span>
                     </div>
-                    <div class="h4" th:text="#{('report.iab.title.grade.' + ${report.studentEnrollment.gradeCode})(${#strings.toString(exam.schoolYear)}, ${support.getLastTwoDigits(exam.schoolYear + 1)})}"></div>
+                    <div class="h4" th:text="#{('report.iab.title.grade.' + ${report.assessment.gradeCode})(${#strings.toString(exam.schoolYear - 1)}, ${support.getLastTwoDigits(exam.schoolYear)})}"></div>
```
Comment (esotrope) [inline common/src/main/resources/templates/iab-body.html]: went ahead and rolled the school year display fix into this

**Pull request merged**